cmake/module/AddBoostIfNeeded.cmake

```diff
@@ -65,7 +65,7 @@ function(add_boost_if_needed)
   endif()
 
   if(BUILD_TESTS)
-    # Some package managers, such as vcpkg, vendor Boost.Test separately
+    # Some package managers, such as vcpkg, vendor Boost. Test separately
```

Comment:
It's not a typo :)

It refers to the "[Boost.Test](https://www.boost.org/doc/libs/1_86_0/libs/test/doc/html/index.html)" module.